sdk/search/Azure.Search.Documents/src/Indexes/SearchIndexClient.cs

```diff
@@ -802,8 +748,7 @@ public virtual SearchClient GetSearchClient(string indexName)
 
                     Response<ListIndexesResult> result = await IndexesClient.ListAsync(
                         Constants.NameKey,
-                        options?.ClientRequestId,
-                        cancellationToken)
+                            cancellationToken)
```

Comment:
Nit - Maybe the indentation thing was a weird find/replace?

Reply:
Yeah, I was using the regex search/replace. The whole line got selected, so not sure what happened here.